lib/sandbox/execute.js

```diff
@@ -72,23 +86,57 @@ module.exports = function (bridge, glob) {
             module: _module
         });
 
-        scope.exec(template, (err) => {
-            if (err) {
-                return bridge.dispatch('initialize', err);
-            }
+        for (const templateName of Object.keys(templates || {})) {
+            const template = templates[templateName],
+                // eslint-disable-next-line no-await-in-loop, no-loop-func
+                templateExecResult = await new Promise((resolve) => {
+                    scope.exec(template, (err) => {
+                        if (err) {
+                            return resolve({ error: err });
+                        }
+
+                        const templateExports = (_module && _module.exports) || {};
+
+                        templatesRegistry[templateName] = templateExports;
+
+                        if (_.isFunction(templateExports.chaiPlugin) && !chaiPluginInitialized) {
```

Comment:
this should only happen for `template` option and not when chaiPlugin option is explicity provided